api/api/management/commands/sendapimoveannouncement.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+
+from django.conf import settings
+from django.core.mail import send_mail
+
+import django_redis
+from django_tqdm import BaseCommand
+
+from api.models.oauth import OAuth2Verification
+
+
+message = """
+On 3 June 2024 the Openverse API’s home will be api.openverse.org and api.openverse.engineering will start
+redirecting to api.openverse.org. Existing API credentials and all other aspects of integrations with the
+Openverse API will continue to work without changes, provided requests follow redirects.
+
+For the best experience, please update code referencing api.openverse.engineering to use api.openverse.org.
```

Comment:
```suggestion
For the best experience, and to avoid redirects, please update code referencing api.openverse.engineering to use api.openverse.org.
```